src/handlebars/users.handlebars

```diff
@@ -98,6 +98,12 @@
           <img src="https://static1.squarespace.com/static/5a78ccb7dc2b4a2c76787366/t/5a7a2be1652dead5a32a0f8b/1553085704781/?format=500w" alt="Utilus logo" />
         </div>
 
+		<div class="clients">
+          <img src="dist/assets/travelcompositor.png" alt="Travelcompositor logo" />
```

Comment:
is it possible to directly link to this hosted elsewhere? I'd rather avoid locally hosting all the logos.